src/pages/ADN/meta.json

```diff
@@ -0,0 +1,5 @@
+{
+  "urls": {
+    "match": ["*://animationdigitalnetwork.fr/*"]
```

Comment:
The search shortcut url is missing

example: https://github.com/MALSync/MALSync/blob/master/src/pages/An1me/meta.json#L2